Automattic/wp-calypso
# Move the `/titan` incoming redirect page to `/incoming-redirect/titan`

We've discussed that it'll be better if we have all incoming URLs from titan be under some more common path so it's clearer what's happening. The path we chose is `/incoming-redirect/titan` instead of just `/titan`.

#### Changes proposed in this Pull Request

* Move the incoming redirect links from `/titan` to `/incoming-redirect/email/titan`
* Also handle 404s

#### Testing instructions

* Make sure that http://calypso.localhost:3000/incoming-redirect/email/titan/staging shows you the "Redirecting..." screen and "Uh oh. Page not found." message if you enter anything after `incoming-redirect` like http://calypso.localhost:3000/incoming-redirect/random1kl23jkj123/123123123

see: pMz3w-cv1-p2
